test/docs/jdk/javadoc/doccheck/ExtLinksJdk.txt

```diff
@@ -0,0 +1,764 @@
+http://cldr.unicode.org/
```

Comment:
One thing that would be nice to have (and easy to implement) is to treat lines starting with `#` as comments and add a few lines at the top of the file describing the purpose of this file and how to add new links.

Reply:
Fixed, thanks!